frontend-nextjs/Dockerfile

```
@@ -0,0 +1,69 @@
+    fi
+
+# Production image, copy all the files and run next
+FROM base AS runner
+WORKDIR /app
+
+ENV NODE_ENV=production
+# Uncomment the following line in case you want to disable telemetry during runtime.
+# ENV NEXT_TELEMETRY_DISABLED=1
+
+RUN addgroup --system --gid 1001 nodejs
+RUN adduser --system --uid 1001 nextjs
```

Hadolint

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)